python/foglamp/plugins/south/coap/coap.py

```diff
@@ -108,8 +110,8 @@ def plugin_start(handle):
     Returns:
     Raises:
     """
-    global _task, loop
-    _LOGGER.info("plugin_start called")
+    global _task, loop, t
+    _LOGGER.debug("plugin_start called")
```

Comment:
I was in favor to remove such comments and test patched calls

Reply:
Such debug logs help in code coverage whereas method patching bypasses method body. Hence debug logs are preferred. Patching should be used only for those methods which play a secondary role in the method under testing, e.g. logger.